### ISSUE TITLE
Upgrade to Svelte v4

### ISSUE THREAD

**Issue by umaranis** [umaranis/svelte-lexical#69]

**Comment by FarhanAliRaza** [comment]
how can i help?

**Comment by enesbala5** [comment]
@umaranis Was this fixed in release 0.3.5? 

It is still not working properly with Sveltekit 2.0 setup, as I am getting this error:

```
SyntaxError: [vite] Named export 'registerPlainText' not found. The requested module '@lexical/plain-text' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export...
```

Thanks for the support on the package once again 🙏

**Comment by umaranis** [comment]
Hi @enesbala5 , 
The last release upgrades to svelte 4. 
But it is not upgraded to SvelteKit 2 yet. Unfortunately, there is a blocker to upgrading to SvelteKit 2 due to its dependency on Vite 5 which in turn doesn’t work well with lexical CJS module at the moment.
